bindings/src/main/java/io/pravega/storage/filesystem/FileSystemStorage.java

```diff
@@ -187,6 +189,37 @@ public boolean supportsTruncation() {
         return false;
     }
 
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(value = "OS_OPEN_STREAM", justification = "Rare operation. The leaked object is collected by GC. In case of a iteraror in a for loop this would be fast.")
+    @Override
+    public Iterator<SegmentProperties> listSegments() {
+        if (log.isDebugEnabled()) {
```

Comment:
I don't think we need to check isDebugEnabled

Reply:
Done.